src/routers/library/library.py

```diff
@@ -1,147 +1,83 @@
-        dict: A message confirming the successful addition of the new patrons.
+    Add new patrons to the library asynchronously.
     """
-    library.add_new_patron_to_the_library(patrons_to_add=add_new_patrons_request.patrons_to_add)
+    await library.add_new_patron_to_the_library(patrons_to_add=add_new_patrons_request.patrons_to_add)
     return {
         "message": f"The new patrons {add_new_patrons_request.patrons_to_add} successfully added"
     }
 
 
 @library_router.delete(path="/patrons/{id}", response_model=LibraryPatronStatusResponse, tags=["Library Patrons"])
-def delete_patron(patron_id: str, library=Depends(get_library)):
+async def delete_patron(patron_id: str, library=Depends(get_library)):
     """
-    Delete a patron from the library.
-
-    This endpoint allows you to remove a patron from the library's system using their ID.
-
-    Args:
-        patron_id (str): The ID of the patron to be removed.
-        library: The library instance, injected via dependency.
-
-    Returns:
-        dict: A message confirming the successful removal of the patron, along with the patron's ID.
+    Delete a patron from the library asynchronously.
     """
-    library.remove_patron(patron_id=patron_id)
+    await library.remove_patron(patron_id=patron_id)
     return {
         "message": "Patron successfully removed",
         "patron id": patron_id
     }
 
 
 @library_router.delete(path="/items/{item_isbn}", response_model=LibraryItemStatusResponse, tags=["Library Items"])
-def delete_item(item_isbn: str, library=Depends(get_library)):
+async def delete_item(item_isbn: str, library=Depends(get_library)):
     """
-    Delete an item from the library.
-
-    This endpoint allows you to remove an item from the library's collection using its ISBN.
-
-    Args:
-        item_isbn (str): The ISBN of the item to be removed.
-        library: The library instance, injected via dependency.
-
-    Returns:
-        dict: A message confirming the successful removal of the item, along with the item's ISBN.
+    Delete an item from the library asynchronously.
     """
-    library.remove_item(item_isbn_to_remove=item_isbn)
+    await library.remove_item(item_isbn_to_remove=item_isbn)
     return {
         "message": "Item successfully removed from the library",
         "isbn": item_isbn
     }
 
 
 @library_router.get(path="/items/{isbn}", response_model=LibraryItemResponse, tags=["Library Items"])
-def get_item(item_isbn: str, library=Depends(get_library)):
+async def get_item(item_isbn: str, library=Depends(get_library)):
     """
-    Get details of a library item.
-
-    This endpoint retrieves the details of a specific library item using its ISBN.
-
-    Args:
-        item_isbn (str): The ISBN of the item to retrieve.
-        library: The library instance, injected via dependency.
-
-    Returns:
-        dict: A dictionary containing the details of the requested library item.
+    Get details of a library item asynchronously.
```

Comment:
Stop adding asynchronously to every comment.
I know it is async sync you used `async def`